StartScreen/Javascript.js

```diff
@@ -1,23 +1,40 @@
 function region(){
-document.getElementsByClassName("regionSelect")[1].click();	}
+document.getElementsByClassName("regionSelect")[1].click();	
+}
  region();
 function tabs(){
-document.getElementsByClassName("strong tab")[0].click();		}
+document.getElementsByClassName("strong tab")[0].click();		
+}
 tabs();
 function get(){
-document.getElementsByClassName("os_block  win7")[0].click();   }
+document.getElementsByClassName("os_block  win7")[0].click();  
+ }
 get();
 function vm(){
-document.getElementsByClassName("vmType")[2].click();		}
+document.getElementsByClassName("vmType")[2].click();		
+}
 vm();
 function checkmark(){
-document.getElementsByClassName("circleOpt")[2].click();  }
+document.getElementsByClassName("circleOpt")[2].click();  
+}
 checkmark();
 function final(){
-document.getElementsByClassName("hollow medium whiteGlo")[0].click();  }
+document.getElementsByClassName("hollow medium whiteGlo")[0].click(); 
+ }
 final();
-//Make the machine verify
- if (confirm("On/Ready")) {
-  txt = "VM is running!";
+function ready() {
+	document.getElementsByClassName("state-on")[1].innerText.includes("On/Ready");
+	return (true);
 }
+ready();
+ //if (ready === true ){setTimeout(function(){
+    window.location.href = 'https://www.paperspace.com/console/machines';
+    }, 120000);}
+    
+    
+    // setTimeout(function(){
+    window.location.href = 'https://www.paperspace.com/console/machines';

```

Comment:
I was a little confused why the function was repeated twice

Reply:
I think it was accidentally copied twice.